.github/CODEOWNERS

```diff
@@ -1,75 +1 @@
-# required for merging a PR (depends on repository settings).
-#
-# For details about syntax, see:
-# https://help.github.com/en/articles/about-code-owners
-# But here are some important notes:
-#
-# - Glob syntax is git-like, e.g. `/core` means the core directory in the root, unlike `core`
-#   which can be everywhere.
-# - Multiple owners are supported.
-# - Either handle (e.g, @github_user or @github/team) or email can be used. Keep in mind,
-#   that handles might work better because they are more recognizable on GitHub,
-#   eyou can use them for mentioning unlike an email.
-# - The latest matching rule, if multiple, takes precedence.
-
-# CI
-/.github/ @paritytech/ci @paritytech/release-engineering
-/.gitlab-ci.yml @paritytech/ci
-/.gitlab/ @paritytech/ci
-
-# XCM
-/polkadot/xcm/ @paritytech/xcm
-
-# WASM executor, low-level client <-> WASM interface and other WASM-related code
-/substrate/client/allocator/ @koute
-/substrate/client/executor/ @koute
-/substrate/primitives/panic-handler/ @koute
-/substrate/primitives/runtime-interface/ @koute
-/substrate/primitives/wasm-interface/ @koute
-/substrate/utils/wasm-builder/ @koute
-
-# Systems-related bits and bobs on the client side
-/substrate/client/sysinfo/ @koute
-/substrate/client/tracing/ @koute
-
-# Documentation audit
-/substrate/primitives/runtime @paritytech/docs-audit
-/substrate/primitives/arithmetic @paritytech/docs-audit
-# /primitives/core (to be added later)
-# /primitives/io (to be added later)
-
-# FRAME
-/substrate/frame/ @paritytech/frame-coders @paritytech/docs-audit
-/substrate/frame/nfts/ @jsidorenko @paritytech/docs-audit
-/substrate/frame/state-trie-migration/ @paritytech/frame-coders @cheme
-/substrate/frame/uniques/ @jsidorenko @paritytech/docs-audit
-
-# GRANDPA, BABE, consensus stuff
-/substrate/client/consensus/babe/ @andresilva
-/substrate/client/consensus/grandpa/ @andresilva
-/substrate/client/consensus/pow/ @sorpaas
-/substrate/client/consensus/slots/ @andresilva
-/substrate/frame/babe/ @andresilva
-/substrate/frame/grandpa/ @andresilva
-/substrate/primitives/consensus/pow/ @sorpaas
-
-# BEEFY, MMR
-/substrate/frame/beefy/ @acatangiu
-/substrate/frame/beefy-mmr/ @acatangiu
-/substrate/frame/merkle-mountain-range/ @acatangiu
-/substrate/primitives/merkle-mountain-range/ @acatangiu
-
-# Contracts
-/substrate/frame/contracts/ @athei @paritytech/docs-audit
-
-# NPoS and election
-/substrate/frame/election-provider-multi-phase/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/election-provider-support/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/elections-phragmen/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/nomination-pools/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/frame/staking/ @paritytech/staking-core @paritytech/docs-audit
-/substrate/primitives/npos-elections/ @paritytech/staking-core @paritytech/docs-audit
+bridges/snowbridge/*.rs @snowfork/snowbridge-devs
```

Comment:
Looks like github won't allow external code owners.

Should probably ask Parity if they can add us as limited permission guests to their github org, so they can create a codeowner group.

Reply:
For the paritytech/polkadot-sdk, yes. For our repo, we just need to make our snowbridge-devs team public. Do you have access to do that? 

See attached images.

![Screenshot 2024-02-06 at 08 26 57](https://github.com/Snowfork/polkadot-sdk/assets/1788149/fbea6a7f-f2ae-4563-8c9a-b6b7338463df)
![Screenshot 2024-02-06 at 08 27 58](https://github.com/Snowfork/polkadot-sdk/assets/1788149/bcf4f3c0-ea49-4f2e-81d1-8c10e6eeee73)

Reply:
Ah ok, I made the team public

Reply:
Thanks, the CODEOWNERS file is now marked as valid. 👍🏻